src/arrayMethodShift.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomShift() {
   [].__proto__.shift2 = function() {
-    // write code here
+    const firstElement = this[0];
+
+    for (let i = 1; i < this.length; i++) {
+      this[i - 1] = this[i];
+    }
+
+    this.length !== 0 ? this.length = this.length - 1 : this.length = 0;
```

Comment:
before writing ternary operator you need to specify the element you're working with
`this.length = ...`